gen/generate.go

```diff
@@ -165,6 +165,36 @@ func Generate(m *compile.Module, o *Options) error {
 	return nil
 }
 
+func duplicateAfterNormalization(fileName string) error {
+	absFileName, err := filepath.Abs(fileName)
+	if err != nil {
+		return generateError{
+			Name:   fileName,
+			Reason: fmt.Errorf("File %q does not exist: %v", fileName, err),
```

Comment:
1. [error strings should start with lower case](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
2. This error message isn't accurate. `filepath.Abs` could fail for a different reason, not just because the file doesn't exist. I would use `Reason: err` here directly.
3. According to the [compile.Module docs](https://godoc.org/go.uber.org/thriftrw/compile#Module), ThriftPath is known to be the absolute path to the Thrift file so `filepath.Abs` on it is unnecessary.

Reply:
done